src/Pages/Main/Components/CategoriesBar/CategoriesBar.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import './CategoriesBar.scss';
+
+export default class CategoriesBar extends Component {
+  constructor() {
+    super();
+    this.state = {
+    categoryTitle: [],
+    };
+  }
```

Comment:
```suggestion
    state = {
      categoryTitle: [],
    };
```
이렇게 할당해주시면 contructor구조를 생략해주실 수 있습니다.

Reply:
수정했습니다! 감사합니다!!